doc/internal/man3/evp_generic_fetch.pod

```diff
@@ -45,6 +54,11 @@ This is meant to be used when one method needs to fetch an associated
 other method, and is typically called from inside the given function
 I<new_method>.
 
+evp_generic_fetch_from_prov() does the same thing as evp_generic_fetch(),
+but limits the search of methods to the provider given with I<prov>.
+This is meant to be used when one method needs to fetch an associated
+other method in the same provider.
```

Comment:
Drop the _other_.  I think it is implied.

Reply:
Done